traits/MLControl.php

```diff
@@ -143,6 +143,10 @@ public function loadLocaleAssets()
      */
     public function getLocaleValue($locale)
     {
+        if (!method_exists($this->model, 'methodExists')) {
+            return $this->formField->value;
+        }
```

Comment:
Why is this required and what is this doing?

Reply:
This prevents error when the model linked to the MLControl based formfield is not an extendable model (like RainLab\Pages\Classes\MenuItem)

Reply:
That's not a good way of handling it. Try this instead:
```php
$methodExists = function ($object, $method) {
    if (method_exists($object, 'methodExists') {
        return $object->methodExists($method);
    } else {
        return method_exists($object, $method);
    }
};

// replace $this->model->methodExists() calls with $methodExists($model, $method);
```

Reply:
Then I should make that a method of the MLControll trait... that way I wouldn't have to define this multiple times...